src/components/Dashboard/LegalObligation.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div v-if="showLegalObligation && isBusinessWithNoMaintenanceFilings" class="legal-obligation-container">
+  <div v-if="showLegalObligation && isBusinessWithNoMaintenanceFilings && !tempRegNumber"
```

Comment:
Good fix. Please mention it in one of the tickets for Riyaz to see. (I sometimes add a "Test Notes" section in the ticket to suggest things to verify.)

Reply:
Sure. Updated ticket 3828